WordPress/Classes/ViewRelated/NUX/SiteCreation.storyboard

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The Domain Suggestion nav bar doesn't match the design:
- The Back button should be "Back".
- The title should be "Create New Site".